beets/library.py

```diff
@@ -468,6 +468,9 @@ class Item(LibModel):
         'discogs_albumid': types.INTEGER,
         'discogs_artistid': types.INTEGER,
         'discogs_labelid': types.INTEGER,
+        'vgmdb_album_id': types.INTEGER,
+        'vgmdb_artist_id': types.INTEGER,
+        'vgmdb_game': types.STRING,
```

Comment:
There's also no need to add fixed attributes for the new fields. Let's just keep them as flexible attributes (meaning that the plugin, not `library.py`, will provide the types).